### DISCUSSION TITLE
Quantized redshifts in quasars and galaxies

### DISCUSSION THREAD

**Discussion by ExpEarth** [a-cosmology-group/acg#270] [Redshift Models]
A new paper has come out on this topic

https://iopscience.iop.org/article/10.1088/1674-4527/ad6db5

and a story by the authors also appeared:

https://phys.org/news/2024-09-quantized-redshift-big-hypothesis.html

However, things have become a lot more complicated since Arp's time and the hypothesis seems to have lost its original clarity.  It's actually really hard to grasp what they are even saying.  For example, what definition of a quasar are they using?  Is it Arp's idea of a body ejected from a galaxy or is it the modern one, where the quasar is part of the supermassive black hole that most galaxies have?  Can someone digest this and summarize?

**Comment by LifeIsStrange** [comment]
I was about to share this study, IMO it is remarkable since it has been ~a decade since the last paper on redshift quantization despite being empirically reproduced many times.
And why are they using such an old version of SDSS (DR7?)

it would be nice if there was a script that could automatically quantify redshift quantization in any standard photometric dataset of galaxies and quasars.
Nowadays it could be done on decaps 2 or panstarrs instead of SDSS.
